ingest/src/main/java/com/microsoft/azure/kusto/ingest/source/BlobSourceInfo.java

```diff
@@ -66,4 +64,26 @@ public Map<String, String> getTracingAttributes() {
         }
         return attributes;
     }
+
+    /*
+     * For internal usage, adding blobExactSize
+     */
+    public static BlobSourceInfo fromFile(String blobPath, FileSourceInfo fileSourceInfo, CompressionType sourceCompressionType, boolean gotCompressed) {
```

Comment:
If it's for internal use - can we lower the visibility?

Reply:
no because its used outside the pacakge

Reply:
Ok, so maybe we want to document it like a public function?

Not sure what the right call is ehre